generator/views/glean_ping_view.py

```diff
@@ -294,6 +295,24 @@ def _make_dimension(
             ],
         }
 
+        if lookml["type"] == "time":
+            # Remove any _{type} suffix from the dimension group name because each timeframe
+            # will add a _{type} suffix to its individual dimension name.
+            lookml["name"] = re.sub("_(date|time(stamp)?)$", "", looker_name)
```

Comment:
I don't currently see any metric columns that end in `_datetime`, so I don't think that's essential for this PR.

In general I do agree we should also remove `_datetime` suffixes like we do for `_date`, `_time`, and `_timestamp`, but that should be done consistently for all columns, and there are currently some columns with `_datetime` suffixes in `fenix.attributable_clients_v2` and `fenix.firefox_android_clients`, so that would be a potentially breaking change for those which I'd don't want to hassle with in this PR.

If you'd like to create a separate PR for that I'd `r+` it (just make sure to also update the [equivalent code in `lookml_utils.py`](https://github.com/mozilla/lookml-generator/blob/7a386b47097030f7876361e45ce374314053a93f/generator/views/lookml_utils.py#L67)).